tech/database/redis/about.md

```diff
@@ -53,7 +48,7 @@ $ sudo systemctl restart redis
 Secure redis by enabling authentication. Add this line in `/etc/redis/redis.conf`
 
 ```console
-$ requirepass  <AuthPassword>
```

Comment:
Just to be sure: do I understand correctly that this is the line user should add to the file above?